### DISCUSSION TITLE
Role-based access control

### DISCUSSION THREAD

**Discussion by GHAFRI** [outline/outline#5910] [General] [answered]
I'm looking for a feature that allows for role-based content visibility within a single page. Specifically, I want to have sections of content on the same page that are conditionally displayed based on the logged-in user's role. For instance, a "support" role user should be able to see both "guest" and "support" sections, while a "mod" role user should see "guest", "support", and "mod" sections, all on the same page. This is not about restricting access to entire pages based on roles, but rather displaying or hiding specific sections of content within a page. Does your platform offer such functionality natively, or is there a way to achieve this using custom code?

**Comment by tommoor** [comment, marked as answer]
There isn’t a way to achieve this and I’m afraid I don’t see a future where we build this functionality

**Comment by Limezy** [comment]
You could make internal links to documents that have their own access rights I guess ?
